src/main/java/seedu/address/model/person/LastContact.java

```diff
@@ -49,17 +49,15 @@ public static boolean isValidDateTime(String dateTime) {
             return true;
         }
         String trimmedDateTime = dateTime.trim();
-        DateTimeFormatter formatter = DateTimeFormatter.ofPattern("dd-MM-uuuu HHmm")
-                .withResolverStyle(ResolverStyle.STRICT);
         try {
-            LocalDateTime.parse(trimmedDateTime, formatter);
```

Comment:
@Colex2000 please take note, i itchy finger and went to replace ur DateTimeFormatter here to the one you have already initiated as a CONST above

Reply:
Thanks, mb I forgot to update.